content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Feature onboarding
+---
+
+import {Box, Flex, BorderBox, Text} from '@primer/components'
+
+This is a guide for designing onboarding for the product and does not include what to do for marketing pages, email announcements, social media, etc.
+
+Onboarding is a virtual unboxing experience that helps users get started with an app.
+
+- ✅ Think creatively utilizing different ways to onboard users, such as empty states and off-product avenues.
```

Comment:
Can we have a sentence that introduces this list? Is it like: "When designing a feature onboarding flow, you should:"